resize.c

```diff
@@ -42,50 +43,95 @@
 #endif
 #endif
 
-/**
- * mutt_resize_screen - Called after receiving SIGWINCH
- */
-void mutt_resize_screen(void)
+int mutt_resize_getrows(int rows)
```

Comment:
We don't need the `rows` parameter.
The old (invalid) value we're passed is just overwritten anyway.